setup.cfg

```diff
@@ -1,2 +1,5 @@
 [pytest]
 addopts = -s --cov=easysnmp --cov-report=term-missing --flake8
+
+[flake8]
+exclude = .svn,CVS,.bzr,.hg,.git,__pycache,build
```

Comment:
Ah, good catch. I copied straight from manpage of flake8, but partially selected __pycache__.